samples/caf/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -40,30 +40,78 @@
 
 &pwm0 {
 	status = "okay";
-	ch0-pin = <13>;
-	ch0-inverted;
+	pinctrl-0 = <&pwm0_default>;
+	pinctrl-1 = <&pwm0_sleep>;
+	pinctrl-names = "default", "sleep";
 };
 
 &pwm1 {
 	status = "okay";
-	ch0-pin = <14>;
-	ch0-inverted;
+	pinctrl-0 = <&pwm1_default>;
+	pinctrl-1 = <&pwm1_sleep>;
+	pinctrl-names = "default", "sleep";
 };
 
 &pwm2 {
 	status = "okay";
-	ch0-pin = <15>;
-	ch0-inverted;
+	pinctrl-0 = <&pwm2_default>;
+	pinctrl-1 = <&pwm2_sleep>;
+	pinctrl-names = "default", "sleep";
 };
 
 &pwm3 {
 	status = "okay";
-	ch0-pin = <16>;
-	ch0-inverted;
+	pinctrl-0 = <&pwm3_default>;
+	pinctrl-1 = <&pwm3_sleep>;
+	pinctrl-names = "default", "sleep";
 };
 
 &pwm_led0 {
 	status = "okay";
 	pwms = <&pwm0 13>;
 	label = "LED 1";
 };
+
+&pinctrl {
+	pwm1_default: pwm1_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT1, 0, 14)>;
```

Comment:
It should be `PWM_OUT0`, for all PWM instances, as originally only `ch0-*` properties were used.

Reply:
Same here.